composer.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
is this supposed to be here?

Reply:
Oops, no thats me trying to install stuff in the root. I'll drop.